composer.json

```diff
@@ -14,41 +14,26 @@
         "symfony/messenger": "^4.4 || ^5.0",
         "thecodingmachine/safe": "^1.0"
     },
+    "conflict": {
+        "setono/post-nord-php-sdk": "<1.1.3"
+    },
     "require-dev": {
-        "behat/behat": "^3.6",
-        "behat/mink": "^1.8",
-        "behat/mink-browserkit-driver": "^1.3",
-        "behat/mink-extension": "^2.3",
-        "behat/mink-selenium2-driver": "^1.4",
-        "ergebnis/composer-normalize": "^2.4",
-        "friends-of-behat/page-object-extension": "^0.3",
-        "friends-of-behat/suite-settings-extension": "^1.0",
-        "friends-of-behat/symfony-extension": "^2.0",
-        "friends-of-behat/variadic-extension": "^1.1",
         "kalessil/production-dependencies-guard": "dev-master",
-        "korbeil/phpstan-generic-rules": "^0.2.2",
         "kriswallsmith/buzz": "^1.0",
-        "lakion/mink-debug-extension": "^1.2.3",
         "matthiasnoback/symfony-dependency-injection-test": "^4.1",
         "nyholm/psr7": "^1.1",
         "phpspec/phpspec": "^6.0",
-        "phpstan/extension-installer": "^1.0",
-        "phpstan/phpstan": "^0.12",
-        "phpstan/phpstan-doctrine": "^0.12",
-        "phpstan/phpstan-strict-rules": "^0.12",
-        "phpstan/phpstan-webmozart-assert": "^0.12",
         "phpunit/phpunit": "^8.5",
         "roave/security-advisories": "dev-master",
+        "setono/code-quality-pack": "^1.3",
         "setono/dao-bundle": "^1.0",
         "setono/gls-webservice-bundle": "^1.0.1",
         "setono/post-nord-bundle": "^1.0.1",
-        "sylius-labs/coding-standard": "^3.1",
-        "symfony/browser-kit": "^4.4 || ^5.0",
+        "setono/sylius-behat-pack": "^0.1.0",
         "symfony/debug-bundle": "^4.4 || ^5.0",
         "symfony/dotenv": "^4.4 || ^5.0",
         "symfony/web-profiler-bundle": "^4.4 || ^5.0",
-        "symfony/web-server-bundle": "^4.4 || ^5.0",
-        "thecodingmachine/phpstan-safe-rule": "^1.0"
+        "symfony/web-server-bundle": "^4.4 || ^5.0"
```

Comment:
This is deprecated. Use Symfony CLI instead. If you want to add it to the build, you can do it under `tools` under the setup php. Like this: https://github.com/shivammathur/setup-php#wrench-tools-support

Reply:
Thanks for the idea :)